### PR TITLE
Vmanomaly Guide - dashboard and query change

### PR DESCRIPTION
**Pull request by Cambaza**
- dashboard fixes
- query change
- minor fixes

### CODE REVIEW

File: deployment/docker/vmanomaly/vmanomaly-integration/README.md [hunk 1]
```diff
@@ -2,14 +2,14 @@
 
 Please read the "vmanomaly integration" guide first - [https://docs.victoriametrics.com/anomaly-detection/guides/guide-vmanomaly-vmalert.html](https://docs.victoriametrics.com/anomaly-detection/guides/guide-vmanomaly-vmalert.html)
 
-To make this Docker compose file work, you MUST replace the content of [vmanomaly_license.txt](https://github.com/VictoriaMetrics/VictoriaMetrics/tree/master/deployment/docker/vmanomaly/vmanomaly-vmalert-guide/vmanomaly_license.txt) with valid license.
+To make this Docker compose file work, you MUST replace the content of [vmanomaly_license](https://github.com/VictoriaMetrics/VictoriaMetrics/tree/master/deployment/docker/vmanomaly/vmanomaly-vmalert-guide/vmanomaly_license) with valid license.
```
Comment (tenmozes) [inline deployment/docker/vmanomaly/vmanomaly-integration/README.md:5]: not found

File: deployment/docker/vmanomaly/vmanomaly-integration/vmanomaly_config.yml [hunk 2]
```diff
@@ -12,7 +12,7 @@ reader:
   datasource_url: "http://victoriametrics:8428/"
   sampling_period: "60s" 
   queries:
-    node_cpu_rate: "quantile by (mode) (0.5, rate(node_cpu_seconds_total[5m]))"
+    node_cpu_rate: "sum(rate(node_cpu_seconds_total[5m])) by (mode, instance,job)"
```
Comment (tenmozes) [inline deployment/docker/vmanomaly/vmanomaly-integration/vmanomaly_config.yml:15]: please prettify the query (add space)

File: deployment/docker/vmanomaly/vmanomaly-integration/vmanomaly_guide_dashboard.json [hunk 3]
```diff
@@ -40,7 +40,7 @@
           "showLineNumbers": false,
           "showMiniMap": false
         },
-        "content": "If you don't see any data, please wait a few minutes. \n\nYou will see a lot of false positive anomalies when you run the guide for the first time. \nThe prediction must be more accurate if you provide vmanomaly 2w of data.\n\nEvery row represents information for one specific mode. \nThe query for anomaly detection is `quantile by (mode) (0.5, rate(node_cpu_seconds_total[5m]))`\nThis is a median (or 50% quantileof `rate` function over `node_cpu_seconds_total`)",
+        "content": "If you don't see any data, please wait a few minutes. \n\nYou will see a lot of false positive anomalies when you run the guide for the first time. \nThe prediction must be more accurate if you provide vmanomaly 2w of data.\n\nEvery row represents information for one specific mode. \nThe query for anomaly detection is `sum(rate(node_cpu_seconds_total)) by (mode)`\n",
```
Comment (fred-navruzov) [inline deployment/docker/vmanomaly/vmanomaly-integration/vmanomaly_guide_dashboard.json:43]: Let's use better wording here, something like:

> "If you don't observe any data initially, please wait a few minutes for it to appear.
> Upon first running the guide (if there is not enough node_exporter monitoring data collected in your system), you may notice a significant number of false positive anomalies found. The predictions will become more accurate with at least two weeks' (full `fit_window`) worth of data provided to vmanomaly.

> Each row displays information for a distinct mode. The query used for anomaly detection is sum(rate(node_cpu_seconds_total)) by (mode)."

Also, it looks like a discrepancy:
in vmanomaly config we use
```markdown
queries:
    node_cpu_rate: "sum(rate(node_cpu_seconds_total[5m])) by (mode, instance,job)"
```
while in this paragraph we use `sum(rate(node_cpu_seconds_total)) by (mode)` (different group by clause)

File: docs/anomaly-detection/guides/guide-vmanomaly-vmalert.md [hunk 4]
```diff
@@ -101,7 +101,7 @@ node_cpu_seconds_total{cpu="1",mode="iowait"} 51.22
...
+The `node_cpu_seconds_total` metric is classified as a [counter](https://docs.victoriametrics.com/keyConcepts.html#counter) type. To analyze the duration each CPU core spends in these modes, it is necessary to compute the rate of change per second using the [rate function](https://docs.victoriametrics.com/MetricsQL.html#rate): `rate(node_cpu_seconds_total)`. For a more refined and smoother aggregation of data by mode, we apply the sum function. The resulting query is formulated as follows: `sum(rate(node_cpu_seconds_total[5m])) by (mode, instance,job)`.
 
 Below is an illustrative example of how this query might be visualized in Grafana:
```
Comment (fred-navruzov) [inline docs/anomaly-detection/guides/guide-vmanomaly-vmalert.md:106]: also, I assume we need to make new screenshot, as now it's about `sum`, not `median`:
image

**Review by fred-navruzov — APPROVED** [review]
Thanks, Daria! LGTM except for a couple of things to be changed (comments left)

**Review by tenmozes — APPROVED** [review]
LGTM but please fix the other comments, thank you!

File: deployment/docker/vmanomaly/vmanomaly-integration/vmanomaly_guide_dashboard.json (hunk 3 above)
Comment (Cambaza) [reply deployment/docker/vmanomaly/vmanomaly-integration/vmanomaly_guide_dashboard.json]: Changed the text
The discrepancy was made on purpose to simplify the header. But I changed it to an actual query now

**Pull request merged**